internal/controllers/utils/utils.go

```diff
@@ -480,6 +483,14 @@ func GetServerArgs(inventory *inventoryv1alpha1.Inventory, serverName string) (r
 		return
 	}
 
+	// HwMgr Plugin Controller
+	if serverName == HardwarePluginManagerServerName {
+		result = slices.Clone(HardwarePluginManagerArgs)
+		// Add OAuth command line arguments
+		result = addArgsForOAuth(inventory, result)
```

Comment:
Is this going to cause the process to fail to start when OAuth attributes are added to the Inventory CR because it will add arguments here that the process doesn't support?